fmriprep_rodents/patch/workflows/anatomical.py

```diff
@@ -0,0 +1,420 @@
+    debug : :obj:`bool`
+        Enable debugging outputs
+    name : :obj:`str`, optional
+        Workflow name (default: anat_preproc_wf)
+    skull_strip_mode : :obj:`str`
+        Determiner for T1-weighted skull stripping (`force` ensures skull stripping,
+        `skip` ignores skull stripping, and `auto` automatically ignores skull stripping
+        if pre-stripped brains are detected).
+    skull_strip_fixed_seed : :obj:`bool`
+        Do not use a random seed for skull-stripping - will ensure
+        run-to-run replicability when used with --omp-nthreads 1
+        (default: ``False``).
+    Inputs
+    ------
+    t1w
+        List of T1-weighted structural images
+    t2w
+        List of T2-weighted structural images
+    roi
+        A mask to exclude regions during standardization
+    flair
+        List of FLAIR images
+    subjects_dir
+        FreeSurfer SUBJECTS_DIR
+    subject_id
+        FreeSurfer subject ID
+    Outputs
+    -------
+    t1w_preproc
+        The T1w reference map, which is calculated as the average of bias-corrected
+        and preprocessed T1w images, defining the anatomical space.
+    t1w_brain
+        Skull-stripped ``t1w_preproc``
+    t1w_mask
+        Brain (binary) mask estimated by brain extraction.
+    t1w_dseg
+        Brain tissue segmentation of the preprocessed structural image, including
+        gray-matter (GM), white-matter (WM) and cerebrospinal fluid (CSF).
+    t1w_tpms
+        List of tissue probability maps corresponding to ``t1w_dseg``.
+    std_preproc
+        T1w reference resampled in one or more standard spaces.
+    std_mask
+        Mask of skull-stripped template, in MNI space
+    std_dseg
+        Segmentation, resampled into MNI space
+    std_tpms
+        List of tissue probability maps in MNI space
+    subjects_dir
+        FreeSurfer SUBJECTS_DIR
+    anat2std_xfm
+        Nonlinear spatial transform to resample imaging data given in anatomical space
+        into standard space.
+    std2anat_xfm
+        Inverse transform of the above.
+    subject_id
+        FreeSurfer subject ID
+    t1w2fsnative_xfm
+        LTA-style affine matrix translating from T1w to
+        FreeSurfer-conformed subject space
+    fsnative2t1w_xfm
+        LTA-style affine matrix translating from FreeSurfer-conformed
+        subject space to T1w
+    surfaces
+        GIFTI surfaces (gray/white boundary, midthickness, pial, inflated)
+    See Also
+    --------
+    * :py:func:`~niworkflows.anat.ants.init_brain_extraction_wf`
+    * :py:func:`~smriprep.workflows.surfaces.init_surface_recon_wf`
+    """
+    freesurfer = False
+    workflow = Workflow(name=name)
+    num_t2w = len(t2w)
+    desc = """Anatomical data preprocessing
+: """
+    desc += """\
+A total of {num_t2w} T2-weighted (T2w) images were found within the input
+BIDS dataset."""
+
+    inputnode = pe.Node(
+        niu.IdentityInterface(fields=['t2w', 'roi', 'flair', 'subjects_dir', 'subject_id']),
```

Comment:
Do we not need T1w at all?

```suggestion
        niu.IdentityInterface(fields=['t2w', 'roi']),
```

Reply:
From a selfish perspective, I do have a dataset that uses MP2RAGE T1w images as the structural rather than T2w. This isn't the norm, but I wouldn't say we should rule out T1w completely.

Reply:
I left these fields to play nice with possible upstream connections, but i guess we should just remove them.